app/src/shared/components/Input/Numeric.jsx

```diff
@@ -1,31 +1,86 @@
 // @flow
```

Comment:
@timoxley `TextInput/NumberField` morphed into `Input/Numeric` and went through a face-lifting. May cause conflicts with your recent work. FYI.